nifi-registry/nifi-registry-core/nifi-registry-docker/dockerhub/Dockerfile

```diff
@@ -32,13 +32,17 @@ ENV NIFI_REGISTRY_BINARY_PATH nifi/${NIFI_REGISTRY_VERSION}/${NIFI_REGISTRY_BINA
 ENV NIFI_REGISTRY_BINARY_URL ${MIRROR}/${NIFI_REGISTRY_BINARY_PATH}
 
 ADD sh/ ${NIFI_REGISTRY_BASE_DIR}/scripts/
+RUN chmod -R +x ${NIFI_REGISTRY_BASE_DIR}/scripts/*.sh
 
 # Setup NiFi-Registry user
 RUN groupadd -g ${GID} nifi || groupmod -n nifi `getent group ${GID} | cut -d: -f1` \
     && useradd --shell /bin/bash -u ${UID} -g ${GID} -m nifi \
     && chown -R nifi:nifi ${NIFI_REGISTRY_BASE_DIR} \
     && apt-get update -y \
-    && apt-get install -y curl jq xmlstarlet unzip
+    && apt-get install -y curl jq xmlstarlet unzip \
+    && apt-get -y autoremove \
+    && apt-get clean autoclean \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 USER nifi
 
```

Comment:
Historic inconsistency - this `dockerhub/Dockerfil` uses a `CMD`, but the `nifi-registry`'s `dockermaven/Dockerfile` uses an `ENTRYPOINT`

It's not a big deal, but I think worth being consistent (I spotted this because the way I was trying to test this was to plug it into a local `docker-compose` stack, which currently overrides the `command` so my `yml` works with the image from Docker Hub but not when I build via `dockermaven`

All the `nifi` and `nifi-toolkit` images use `ENTRYPOINT`; `minifi` and `-c2` are all using `CMD` - possibly worth (but maybe not) making them all consistent across the board?